.env.example

```diff
@@ -68,7 +70,7 @@ FROM_EMAIL=from@example.com
 # password resets and others
 FROM_NAME=dyrector.io
 
-## ReCAPTCHA secrets
+# # ReCAPTCHA secrets
```

Comment:
What type of linter / formatter are u using?

Reply:
I'll double check this, if it does it again or I did something esoteric.

Reply:
I'm checked, same here, comment is irrelevant, resolved.